java.lang.NoClassDefFoundError: scala/reflect/api/Symbols$SymbolApi

I'm using the `reformatOnCompileSettings` settings and I got a strange error when I tried to upgrade `sbt-scalafmt` from 0.4.3-RC1 to 0.4.3:

```
>last compile:scalafmt
java.lang.NoClassDefFoundError: scala/reflect/api/Symbols$SymbolApi
	at scala.meta.internal.tokens.package$.XtensionTokenName(package.scala:7)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:416)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:422)
	at scala.meta.internal.parsers.ScalametaParser.inParens(ScalametaParser.scala:326)
	at scala.meta.internal.parsers.ScalametaParser.argumentExprs(ScalametaParser.scala:2060)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1$$anonfun$apply$46.apply(ScalametaParser.scala:1995)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1$$anonfun$apply$46.apply(ScalametaParser.scala:1995)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1.apply(ScalametaParser.scala:1995)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1.apply(ScalametaParser.scala:1977)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.simpleExprRest(ScalametaParser.scala:1977)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExpr$1.apply(ScalametaParser.scala:1974)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExpr$1.apply(ScalametaParser.scala:1943)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.simpleExpr(ScalametaParser.scala:1943)
	at scala.meta.internal.parsers.ScalametaParser.prefixExpr(ScalametaParser.scala:1921)
	at scala.meta.internal.parsers.ScalametaParser.postfixExpr(ScalametaParser.scala:1897)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1$$anonfun$41.apply(ScalametaParser.scala:1594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1$$anonfun$41.apply(ScalametaParser.scala:1594)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1.apply(ScalametaParser.scala:1594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1.apply(ScalametaParser.scala:1522)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.expr(ScalametaParser.scala:1522)
	at scala.meta.internal.parsers.ScalametaParser.expr(ScalametaParser.scala:1499)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$funDefRest$1.apply(ScalametaParser.scala:2997)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$funDefRest$1.apply(ScalametaParser.scala:2969)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.funDefRest(ScalametaParser.scala:2969)
	at scala.meta.internal.parsers.ScalametaParser.funDefOrDclOrSecondaryCtor(ScalametaParser.scala:2965)
	at scala.meta.internal.parsers.ScalametaParser.defOrDclOrSecondaryCtor(ScalametaParser.scala:2909)
	at scala.meta.internal.parsers.ScalametaParser.nonLocalDefOrDcl(ScalametaParser.scala:2883)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3474)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3470)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.templateStats(ScalametaParser.scala:3469)
	at scala.meta.internal.parsers.ScalametaParser.templateStatSeq(ScalametaParser.scala:3456)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.inBraces(ScalametaParser.scala:338)
	at scala.meta.internal.parsers.ScalametaParser.templateBody(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.templateBodyOpt(ScalametaParser.scala:3358)
	at scala.meta.internal.parsers.ScalametaParser.templateOpt(ScalametaParser.scala:3342)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$classDef$1.apply(ScalametaParser.scala:3090)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$classDef$1.apply(ScalametaParser.scala:3074)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.classDef(ScalametaParser.scala:3074)
	at scala.meta.internal.parsers.ScalametaParser.tmplDef(ScalametaParser.scala:3041)
	at scala.meta.internal.parsers.ScalametaParser.defOrDclOrSecondaryCtor(ScalametaParser.scala:2913)
	at scala.meta.internal.parsers.ScalametaParser.nonLocalDefOrDcl(ScalametaParser.scala:2883)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3474)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3470)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.templateStats(ScalametaParser.scala:3469)
	at scala.meta.internal.parsers.ScalametaParser.templateStatSeq(ScalametaParser.scala:3456)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.inBraces(ScalametaParser.scala:338)
	at scala.meta.internal.parsers.ScalametaParser.templateBody(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.templateBodyOpt(ScalametaParser.scala:3358)
	at scala.meta.internal.parsers.ScalametaParser.templateOpt(ScalametaParser.scala:3342)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$objectDef$1.apply(ScalametaParser.scala:3099)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$objectDef$1.apply(ScalametaParser.scala:3097)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.objectDef(ScalametaParser.scala:3097)
	at scala.meta.internal.parsers.ScalametaParser.tmplDef(ScalametaParser.scala:3047)
	at scala.meta.internal.parsers.ScalametaParser.topLevelTmplDef(ScalametaParser.scala:3027)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$topStat$1.applyOrElse(ScalametaParser.scala:3416)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$topStat$1.applyOrElse(ScalametaParser.scala:3406)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.topStatSeq(ScalametaParser.scala:3405)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.scala$meta$internal$parsers$ScalametaParser$$anonfun$$bracelessPackageStats$1(ScalametaParser.scala:3633)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1$$anonfun$apply$87.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1$$anonfun$apply$87.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.apply(ScalametaParser.scala:3611)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:3611)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$source$1.apply(ScalametaParser.scala:3596)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$source$1.apply(ScalametaParser.scala:3595)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:3594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$parseSource$1.apply(ScalametaParser.scala:157)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$parseSource$1.apply(ScalametaParser.scala:157)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:46)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:157)
	at scala.meta.parsers.Parse$$anonfun$parseSource$1.apply(Parse.scala:31)
	at scala.meta.parsers.Parse$$anonfun$parseSource$1.apply(Parse.scala:31)
	at scala.meta.internal.parsers.ScalametaParser$$anon$201.apply(ScalametaParser.scala:3651)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:21)
	at scala.meta.parsers.Api$XtensionParseInputLike.parse(Api.scala:10)
	at org.scalafmt.rewrite.Rewrite$.apply(Rewrite.scala:47)
	at org.scalafmt.Scalafmt$.format(Scalafmt.scala:49)
	at org.scalafmt.cli.Scalafmt210.format(Scalafmt210.scala:46)
	at org.scalafmt.cli.Scalafmt210.format(Scalafmt210.scala:30)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.scalafmt.sbt.HasScalaFmt.org$scalafmt$sbt$HasScalaFmt$$handleFile(HasScalaFmt.scala:85)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.parallel.immutable.ParHashSet$ParHashSetIterator.foreach(ParHashSet.scala:76)
	at scala.collection.parallel.ParIterableLike$Foreach.leaf(ParIterableLike.scala:975)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply$mcV$sp(Tasks.scala:54)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$class.tryLeaf(Tasks.scala:56)
	at scala.collection.parallel.ParIterableLike$Foreach.tryLeaf(ParIterableLike.scala:972)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.internal(Tasks.scala:183)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.internal(Tasks.scala:514)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.compute(Tasks.scala:162)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.compute(Tasks.scala:514)
	at scala.concurrent.forkjoin.RecursiveAction.exec(RecursiveAction.java:160)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.ClassNotFoundException: scala.reflect.api.Symbols$SymbolApi
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at scala.meta.internal.tokens.package$.XtensionTokenName(package.scala:7)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:416)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:422)
	at scala.meta.internal.parsers.ScalametaParser.inParens(ScalametaParser.scala:326)
	at scala.meta.internal.parsers.ScalametaParser.argumentExprs(ScalametaParser.scala:2060)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1$$anonfun$apply$46.apply(ScalametaParser.scala:1995)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1$$anonfun$apply$46.apply(ScalametaParser.scala:1995)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1.apply(ScalametaParser.scala:1995)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExprRest$1.apply(ScalametaParser.scala:1977)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.simpleExprRest(ScalametaParser.scala:1977)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExpr$1.apply(ScalametaParser.scala:1974)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$simpleExpr$1.apply(ScalametaParser.scala:1943)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.simpleExpr(ScalametaParser.scala:1943)
	at scala.meta.internal.parsers.ScalametaParser.prefixExpr(ScalametaParser.scala:1921)
	at scala.meta.internal.parsers.ScalametaParser.postfixExpr(ScalametaParser.scala:1897)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1$$anonfun$41.apply(ScalametaParser.scala:1594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1$$anonfun$41.apply(ScalametaParser.scala:1594)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1.apply(ScalametaParser.scala:1594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1.apply(ScalametaParser.scala:1522)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.expr(ScalametaParser.scala:1522)
	at scala.meta.internal.parsers.ScalametaParser.expr(ScalametaParser.scala:1499)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$funDefRest$1.apply(ScalametaParser.scala:2997)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$funDefRest$1.apply(ScalametaParser.scala:2969)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.funDefRest(ScalametaParser.scala:2969)
	at scala.meta.internal.parsers.ScalametaParser.funDefOrDclOrSecondaryCtor(ScalametaParser.scala:2965)
	at scala.meta.internal.parsers.ScalametaParser.defOrDclOrSecondaryCtor(ScalametaParser.scala:2909)
	at scala.meta.internal.parsers.ScalametaParser.nonLocalDefOrDcl(ScalametaParser.scala:2883)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3474)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3470)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.templateStats(ScalametaParser.scala:3469)
	at scala.meta.internal.parsers.ScalametaParser.templateStatSeq(ScalametaParser.scala:3456)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.inBraces(ScalametaParser.scala:338)
	at scala.meta.internal.parsers.ScalametaParser.templateBody(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.templateBodyOpt(ScalametaParser.scala:3358)
	at scala.meta.internal.parsers.ScalametaParser.templateOpt(ScalametaParser.scala:3342)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$classDef$1.apply(ScalametaParser.scala:3090)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$classDef$1.apply(ScalametaParser.scala:3074)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.classDef(ScalametaParser.scala:3074)
	at scala.meta.internal.parsers.ScalametaParser.tmplDef(ScalametaParser.scala:3041)
	at scala.meta.internal.parsers.ScalametaParser.defOrDclOrSecondaryCtor(ScalametaParser.scala:2913)
	at scala.meta.internal.parsers.ScalametaParser.nonLocalDefOrDcl(ScalametaParser.scala:2883)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3474)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3470)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.templateStats(ScalametaParser.scala:3469)
	at scala.meta.internal.parsers.ScalametaParser.templateStatSeq(ScalametaParser.scala:3456)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.inBraces(ScalametaParser.scala:338)
	at scala.meta.internal.parsers.ScalametaParser.templateBody(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.templateBodyOpt(ScalametaParser.scala:3358)
	at scala.meta.internal.parsers.ScalametaParser.templateOpt(ScalametaParser.scala:3342)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$objectDef$1.apply(ScalametaParser.scala:3099)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$objectDef$1.apply(ScalametaParser.scala:3097)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.objectDef(ScalametaParser.scala:3097)
	at scala.meta.internal.parsers.ScalametaParser.tmplDef(ScalametaParser.scala:3047)
	at scala.meta.internal.parsers.ScalametaParser.topLevelTmplDef(ScalametaParser.scala:3027)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$topStat$1.applyOrElse(ScalametaParser.scala:3416)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$topStat$1.applyOrElse(ScalametaParser.scala:3406)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.topStatSeq(ScalametaParser.scala:3405)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.scala$meta$internal$parsers$ScalametaParser$$anonfun$$bracelessPackageStats$1(ScalametaParser.scala:3633)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1$$anonfun$apply$87.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1$$anonfun$apply$87.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.apply(ScalametaParser.scala:3611)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:3611)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$source$1.apply(ScalametaParser.scala:3596)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$source$1.apply(ScalametaParser.scala:3595)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:3594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$parseSource$1.apply(ScalametaParser.scala:157)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$parseSource$1.apply(ScalametaParser.scala:157)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:46)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:157)
	at scala.meta.parsers.Parse$$anonfun$parseSource$1.apply(Parse.scala:31)
	at scala.meta.parsers.Parse$$anonfun$parseSource$1.apply(Parse.scala:31)
	at scala.meta.internal.parsers.ScalametaParser$$anon$201.apply(ScalametaParser.scala:3651)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:21)
	at scala.meta.parsers.Api$XtensionParseInputLike.parse(Api.scala:10)
	at org.scalafmt.rewrite.Rewrite$.apply(Rewrite.scala:47)
	at org.scalafmt.Scalafmt$.format(Scalafmt.scala:49)
	at org.scalafmt.cli.Scalafmt210.format(Scalafmt210.scala:46)
	at org.scalafmt.cli.Scalafmt210.format(Scalafmt210.scala:30)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.scalafmt.sbt.HasScalaFmt.org$scalafmt$sbt$HasScalaFmt$$handleFile(HasScalaFmt.scala:85)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.parallel.immutable.ParHashSet$ParHashSetIterator.foreach(ParHashSet.scala:76)
	at scala.collection.parallel.ParIterableLike$Foreach.leaf(ParIterableLike.scala:975)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply$mcV$sp(Tasks.scala:54)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$class.tryLeaf(Tasks.scala:56)
	at scala.collection.parallel.ParIterableLike$Foreach.tryLeaf(ParIterableLike.scala:972)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.internal(Tasks.scala:183)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.internal(Tasks.scala:514)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.compute(Tasks.scala:162)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.compute(Tasks.scala:514)
	at scala.concurrent.forkjoin.RecursiveAction.exec(RecursiveAction.java:160)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

@olafurpg , yep,

> scalafmt -version
scalafmt 0.4.3

```
Exception in thread "main" java.lang.NoClassDefFoundError: scala/reflect/api/Symbols$SymbolApi
	at scala.meta.internal.tokens.package$.XtensionTokenName(package.scala:7)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:416)
	at scala.meta.internal.parsers.ScalametaParser.name(ScalametaParser.scala:1188)
	at scala.meta.internal.parsers.ScalametaParser.termName(ScalametaParser.scala:1190)
	at scala.meta.internal.parsers.ScalametaParser.path(ScalametaParser.scala:1226)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$$anonfun$simpleType$1.apply(ScalametaParser.scala:973)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$$anonfun$simpleType$1.apply(ScalametaParser.scala:969)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$class.simpleType(ScalametaParser.scala:969)
	at scala.meta.internal.parsers.ScalametaParser$outPattern$.simpleType(ScalametaParser.scala:2478)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$class.annotType(ScalametaParser.scala:957)
	at scala.meta.internal.parsers.ScalametaParser$outPattern$.annotType(ScalametaParser.scala:2478)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$class.compoundType(ScalametaParser.scala:1007)
	at scala.meta.internal.parsers.ScalametaParser$outPattern$.compoundType(ScalametaParser.scala:2478)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$class.infixType(ScalametaParser.scala:1063)
	at scala.meta.internal.parsers.ScalametaParser$outPattern$.infixType(ScalametaParser.scala:2478)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$$anonfun$typ$1.apply(ScalametaParser.scala:938)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$$anonfun$typ$1.apply(ScalametaParser.scala:935)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$class.typ(ScalametaParser.scala:935)
	at scala.meta.internal.parsers.ScalametaParser$outPattern$.typ(ScalametaParser.scala:2478)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$$anonfun$typ$1.apply(ScalametaParser.scala:941)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$$anonfun$typ$1.apply(ScalametaParser.scala:935)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser$PatternContextSensitive$class.typ(ScalametaParser.scala:935)
	at scala.meta.internal.parsers.ScalametaParser$outPattern$.typ(ScalametaParser.scala:2478)
	at scala.meta.internal.parsers.ScalametaParser.typ(ScalametaParser.scala:2498)
	at scala.meta.internal.parsers.ScalametaParser.typeOrInfixType(ScalametaParser.scala:1458)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1.apply(ScalametaParser.scala:1622)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$expr$1.apply(ScalametaParser.scala:1522)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.expr(ScalametaParser.scala:1522)
	at scala.meta.internal.parsers.ScalametaParser.expr(ScalametaParser.scala:1499)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$funDefRest$1.apply(ScalametaParser.scala:2997)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$funDefRest$1.apply(ScalametaParser.scala:2969)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.funDefRest(ScalametaParser.scala:2969)
	at scala.meta.internal.parsers.ScalametaParser.funDefOrDclOrSecondaryCtor(ScalametaParser.scala:2965)
	at scala.meta.internal.parsers.ScalametaParser.defOrDclOrSecondaryCtor(ScalametaParser.scala:2909)
	at scala.meta.internal.parsers.ScalametaParser.nonLocalDefOrDcl(ScalametaParser.scala:2883)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3474)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateStat$1.applyOrElse(ScalametaParser.scala:3470)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.templateStats(ScalametaParser.scala:3469)
	at scala.meta.internal.parsers.ScalametaParser.templateStatSeq(ScalametaParser.scala:3456)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$templateBody$1.apply(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.inBraces(ScalametaParser.scala:338)
	at scala.meta.internal.parsers.ScalametaParser.templateBody(ScalametaParser.scala:3353)
	at scala.meta.internal.parsers.ScalametaParser.templateBodyOpt(ScalametaParser.scala:3358)
	at scala.meta.internal.parsers.ScalametaParser.templateOpt(ScalametaParser.scala:3342)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$objectDef$1.apply(ScalametaParser.scala:3099)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$objectDef$1.apply(ScalametaParser.scala:3097)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.objectDef(ScalametaParser.scala:3097)
	at scala.meta.internal.parsers.ScalametaParser.tmplDef(ScalametaParser.scala:3047)
	at scala.meta.internal.parsers.ScalametaParser.topLevelTmplDef(ScalametaParser.scala:3027)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$topStat$1.applyOrElse(ScalametaParser.scala:3416)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$topStat$1.applyOrElse(ScalametaParser.scala:3406)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:3388)
	at scala.meta.internal.parsers.ScalametaParser.topStatSeq(ScalametaParser.scala:3405)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.scala$meta$internal$parsers$ScalametaParser$$anonfun$$bracelessPackageStats$1(ScalametaParser.scala:3633)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1$$anonfun$apply$87.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1$$anonfun$apply$87.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.apply(ScalametaParser.scala:3639)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$batchSource$1.apply(ScalametaParser.scala:3611)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:3611)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$source$1.apply(ScalametaParser.scala:3596)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$source$1.apply(ScalametaParser.scala:3595)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:394)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:400)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:3594)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$parseSource$1.apply(ScalametaParser.scala:157)
	at scala.meta.internal.parsers.ScalametaParser$$anonfun$parseSource$1.apply(ScalametaParser.scala:157)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:46)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:157)
	at scala.meta.parsers.Parse$$anonfun$parseSource$1.apply(Parse.scala:31)
	at scala.meta.parsers.Parse$$anonfun$parseSource$1.apply(Parse.scala:31)
	at scala.meta.internal.parsers.ScalametaParser$$anon$201.apply(ScalametaParser.scala:3651)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:21)
	at scala.meta.parsers.Api$XtensionParseInputLike.parse(Api.scala:10)
	at org.scalafmt.Scalafmt$.format(Scalafmt.scala:51)
	at org.scalafmt.cli.Cli$$anonfun$run$1.apply(Cli.scala:218)
	at org.scalafmt.cli.Cli$$anonfun$run$1.apply(Cli.scala:209)
	at scala.collection.parallel.mutable.ParArray$ParArrayIterator.foreach_quick(ParArray.scala:143)
	at scala.collection.parallel.mutable.ParArray$ParArrayIterator.foreach(ParArray.scala:136)
	at scala.collection.parallel.ParIterableLike$Foreach.leaf(ParIterableLike.scala:972)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply$mcV$sp(Tasks.scala:49)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:48)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:48)
	at scala.collection.parallel.Task$class.tryLeaf(Tasks.scala:51)
	at scala.collection.parallel.ParIterableLike$Foreach.tryLeaf(ParIterableLike.scala:969)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.internal(Tasks.scala:159)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.internal(Tasks.scala:443)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.compute(Tasks.scala:149)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.compute(Tasks.scala:443)
	at scala.concurrent.forkjoin.RecursiveAction.exec(RecursiveAction.java:160)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.ClassNotFoundException: scala.reflect.api.Symbols$SymbolApi
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 110 more
```

--- a/bin/issue492.scala
+++ b/bin/issue492.scala
@@ -1,0 +1,3 @@
+object A {
+    <.section(render(props)
+}

--- a/bin/issue492.sh
+++ b/bin/issue492.sh
@@ -1,0 +1,10 @@
+# sbt cli/assembly
+# I can't reproduce this error in sbt, see https://github.com/olafurpg/scalafmt/issues/492
+set -e
+echo "Issue 492"
+in_file=bin/issue492.scala
+out_file=target/issue492-formatted.scala
+sbt cli/assembly
+java -jar cli/target/scala-2.11/scalafmt.jar -f $in_file > $out_file
+diff --ignore-blank-lines $in_file $out_file
+

--- a/bin/testAll.sh
+++ b/bin/testAll.sh
@@ -5,5 +5,6 @@ set -e
 sbt clean test
 sbt "core/test:runMain org.scalafmt.FormatExperimentApp"
 sbt "; publishLocal ; scripted"
+./bin/issue492.sh
 #sbt coverageAggregate
 

--- a/build.sbt
+++ b/build.sbt
@@ -30,7 +30,7 @@ lazy val buildSettings = Seq(
 
 lazy val metaMacroSettings: Seq[Def.Setting[_]] = Seq(
   libraryDependencies += "org.scalameta" %% "scalameta" % Deps.scalameta,
-  sources in (Compile,doc) := Nil,
+  sources in (Compile, doc) := Nil,
   addCompilerPlugin(
     "org.scalameta" % "paradise" % "3.0.0-M5" cross CrossVersion.full),
   scalacOptions += "-Xplugin-require:macroparadise"
@@ -135,12 +135,12 @@ lazy val core = project
       (runMain in Test).toTask(" org.scalafmt.FormatExperimentApp").value
     },
     libraryDependencies ++= Seq(
-      "com.lihaoyi"   %% "sourcecode" % "0.1.2",
-      "org.scalameta" %% "scalameta"  % Deps.scalameta,
-      "com.typesafe"     % "config" % "1.2.1",
+      "com.lihaoyi"    %% "sourcecode"   % "0.1.2",
+      "org.scalameta"  %% "scalameta"    % Deps.scalameta,
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
+      "com.typesafe"   % "config"        % "1.2.1",
       // Test dependencies
       "org.scalariform"                %% "scalariform"    % Deps.scalariform,
-      "org.scala-lang"                 % "scala-reflect"   % scalaVersion.value % "test",
       "org.scala-lang"                 % "scala-compiler"  % scalaVersion.value % "test",
       "ch.qos.logback"                 % "logback-classic" % "1.1.6" % "test",
       "com.googlecode.java-diff-utils" % "diffutils"       % "1.3.0" % "test",

--- a/core/src/test/scala/org/scalafmt/FidelityTest.scala
+++ b/core/src/test/scala/org/scalafmt/FidelityTest.scala
@@ -29,6 +29,7 @@ class FidelityTest extends FunSuite with FormatAssertions {
           "ConfigReader.scala",
           "BuildTime.scala",
           "GitCommit.scala",
+          "issue492.scala",
           "/target/",
           "/resources/",
           "/gh-pages/"

